.github/workflows/e2e.yaml

```diff
@@ -1,54 +1,64 @@
-name: e2e
+name: Elemental End-To-End tests with Rancher Manager
 
 on:
-  push:
+  workflow_run:
```

Comment:
Why this choice of one workflow running the other instead of being jobs in the same workflow?

Reply:
For later re-integration of smoke/integration tests, the `build-ci` process will be the same. So I find this syntax better than duplicating the code. Also the E2E tests will not be triggered on each PR but only on merge on `master` (E2E tests are too long - will be more after re-integration of upgrade tests), buit the build could be to have some "quick" validation of the code.

Reply:
ah interesting, good choice here!